src/pages/what-we-do.pug

```diff
@@ -32,7 +32,7 @@ block pageComponents
 
       +block({
         blockHeading: "Design",
-        blockContent: "<p>Next, we work through each of our recommendations, designing automated solutions to bottlenecks in your business processes. We use <strong>sketching</strong>, <strong>wireframing</strong>, <strong>prototyping</strong>, and <strong>visual design</strong> to communicate a clear vision for how your software will work.</p><p>This allows us to identify and eliminate technological and design hurdles early on, while promoting confidence in the end result from stakeholders, staff, and customers.</p>",
+        blockContent: "<p>We design automated solutions to bottlenecks in your business processes, using <strong>sketching</strong>, <strong>wireframing</strong>, <strong>prototyping</strong>, and <strong>visual design</strong> to communicate a clear vision for how your software will work.</p><p>This allows us to identify and eliminate technological and design hurdles early on, while promoting confidence in the end result from stakeholders, staff, and customers.</p>",
```

Comment:
Second paragraph starts with "This allows us to", but the content should be independent of the paragraph above, and therefore the context needs readdressing.

Potentially something like:  
> **Introducing design-thinking early in the process** allows us to identify...